plugins/containers-plugin/src/containers-plugin.ts

```diff
@@ -18,14 +19,18 @@ import {
 
 import { ContainersService } from './containers-service';
 
-export function start(context: theia.PluginContext): void {
+export async function start(context: theia.PluginContext): Promise<void> {
   const treeDataProvider = new ContainersTreeDataProvider();
   const treeDataDisposableFn = theia.Disposable.create(() => {
     treeDataProvider.dispose();
   });
   context.subscriptions.push(treeDataDisposableFn);
 
-  theia.window.createTreeView('workspace', { treeDataProvider });
+  const view = theia.window.createTreeView('workspace-view', { treeDataProvider });
+  const workspace = await cheTheia.workspace.getCurrentWorkspace();
+  if (workspace.devfile && workspace.devfile.metadata && workspace.devfile.metadata.name) {
+    view.title = workspace.devfile.metadata.name;
```

Comment:
This name is user-defined. Do we have to worry about HTML-encoding this name or is it safe?